ui/helpers/url.js

```diff
@@ -142,3 +142,7 @@ export const bugzillaBugsApi = function bugzillaBugsApi(api, params) {
   const query = createQueryParams(params);
   return `https://bugzilla.mozilla.org/${api}${query}`;
 };
+
+export const getRevisionTxtUrl = function getRevisionTxtUrl() {
```

Comment:
Should this be a string literal instead of a function? Also, what do you think about removing "txt" from the name (since it's somewhat tied to the implementation rather than what the constant represents)?

eg:
```js
export const deployedRevisionUrl = '/revision.txt';
```

Reply:
Yeah, good call.